test/test_namespace.py

```diff
@@ -97,17 +100,23 @@ def test_n32(self):
             in n3
         )
 
+    @property
```

Comment:
Was `@property` decorator added to avoid running this particular test? Commenting it out currently results in a failed test.